ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -990,11 +990,11 @@ Rectangle {
         width: messageInput.width
         filter: messageInputField.text
         cursorPosition: messageInputField.cursorPosition
-        property: ["name", "nickname", "ensName", "alias"]
+        property: ["nickname", "ensName", "name", "alias"]
```

Comment:
Would be great to move this array to ProfileUtils. It would make the logic related to display names contained in one place.

Motivated by:
> follow the order of properties that ProfileUtils.displayName() uses
elsewhere in the application

Reply:
Yeah but these particular names are valid only here 😐